app/view/Cover.svelte

```diff
@@ -1,79 +1,78 @@
+import { cond, equals } from 'ramda';
+import stores from '../stores/index';
+import logo from '../image/logo.png';
+import Dialog from './component/Dialog.svelte';
+import IdentificationForm from './IdentificationForm.svelte';
+import ChooseGameForm from './ChooseGameForm.svelte';
+import CreateGameForm from './CreateGameForm.svelte';
+import JoinGameForm from './JoinGameForm.svelte';
+import Lobby from './Lobby.svelte';
+import Flow from './component/Flow.svelte';
+import _ from '../util/lens';
+import value from '../util/event';
 
-  const { state } = stores();
-  const username = _.user.name(state);
-  const lobby = _.lobby(state);
+const { state } = stores();
+const username = _.user.name(state);
+const lobby = _.lobby(state);
 
-  async function * cover() {
-    $username = value(yield 'identification');
-    yield* chooseGame();
-  }
+async function * cover() {
+  $username = value(yield 'identification');
+  yield* chooseGame();
+}
 
-  async function * chooseGame() {
-    try {
-      const next = value(yield 'choose-game');
-      yield * cond([
-        [equals('create'), createGame],
-        [equals('join'), joinGame],
-      ])(next);
-    } catch {
-      yield * cover();
-    }
+async function * chooseGame() {
+  try {
+    const next = value(yield 'choose-game');
+    yield * cond([
+      [equals('create'), createGame],
+      [equals('join'), joinGame],
+    ])(next);
+  } catch {
+    yield * cover();
   }
+}
 
-  async function * createGame() {
-    try {
-      const { name, settings } = value(yield 'create-game');
-      yield * gameLobby(name, settings);
-    } catch {
-      yield * chooseGame();
-    }
+async function * createGame() {
+  try {
+    const { name, settings } = value(yield 'create-game');
+    yield * gameLobby(name, settings);
+  } catch {
+    yield * chooseGame();
   }
+}
 
-  async function * joinGame() {
-    try {
-      const name = value(yield 'join-game');
-      yield * gameLobby(name);
-    } catch {
-      yield * chooseGame();
-    }
+async function * joinGame() {
+  try {
+    const name = value(yield 'join-game');
+    yield * gameLobby(name);
+  } catch {
+    yield * chooseGame();
   }
+}
 
-  async function * gameLobby(name, settings) {
-    if (settings) {
-      $lobby = {
-        name,
-        ...settings,
-        players: [{ ...$state.user }],
-      };
-      // TODO: Create game
-    } else {
-      $lobby = { name };
-      // TODO: Join game
-    }
-    try {
-      yield 'lobby';
-    } catch {
-      yield * chooseGame();
-    }
+async function * gameLobby(name, settings) {
+  if (settings) {
+    $lobby = {
+      name,
+      ...settings,
+      players: [{ ...$state.user }],
+    };
+    // TODO: Create game
+  } else {
+    $lobby = { name };
+    // TODO: Join game
+  }
+  try {
+    yield 'lobby';
+  } catch {
+    yield * chooseGame();
   }
+}
 </script>
 
 <div class='cover'>
-  <img src={logo} />
+  <img src={logo} alt="root-cover" />
```

Comment:
I think we don't need alt here. Some `a11y` rules don't make a lot of sense for a decorative image in the background. Maybe this `.cover` should have a `background-image` instead of an `<img>` tag

```suggestion
  <img src={logo} />
```